hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/SimpleCopyListing.java

```diff
@@ -288,10 +288,8 @@ protected void doBuildListingWithSnapshotDiff(
 
           FileStatus sourceStatus = sourceFS.getFileStatus(diff.getTarget());
           if (sourceStatus.isDirectory()) {
-            if (LOG.isDebugEnabled()) {
-              LOG.debug("Adding source dir for traverse: " +
-                  sourceStatus.getPath());
-            }
+            LOG.debug(
+                "Adding source dir for traverse: " + sourceStatus.getPath());
```

Comment:
sorry, I meant you can do the LOG.debug("Adding source dir for traverse: {}", sourceStatus.getPath());  style of log; the existing code is all for commons logging and uses concatenation -so adds the isDebug guards to avoid doing work when debug is false.